JSDemos/Demos/Charts/PointsAggregationFinancialChart/description.md

```diff
@@ -1 +1,5 @@
-This demo illustrates how the **Chart** widget aggregates series points in&nbsp;a&nbsp;financial chart. The **useAggregation** option activates this feature. The **RangeSelector** widget zooms the contents of&nbsp;the **Chart**. Note that whenever the selected range is&nbsp;reduced, the series becomes less aggregated.
\ No newline at end of file
+The **Chart** widget can aggregate series points. To enable this feature, set the **aggregation**.[enabled](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/#enabled) option of a [series](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/) to **true**. 
+
+In this demo, the **Chart** is bound to the [RangeSelector](/Documentation/ApiReference/Data_Visualization_Widgets/dxRangeSelector/) component. When you narrow down the selection range, the chart zooms in, and the series points become less aggregated. The converse is also true: when the selected range is wider, the degree of aggregation is higher.
+
+Aggregation on zoom is only one use case for this feature. Refer to the [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/) description for information on how to configure aggregation for other use cases.
```

Comment:
I didn't want to repeat "cases" in two sentences. "One of many possible" sounds a bit better to me than "only one". The latter is somewhat vague - do we mean "one of many" or "the only one"?

```suggestion
Aggregation on zoom is one of many possible use cases for this feature. Refer to the [aggregation](/Documentation/ApiReference/Data_Visualization_Widgets/dxChart/Configuration/series/aggregation/) description for information on how to configure aggregation for other scenarios.
```